masterlist.yaml

```
@@ -3523,6 +3523,40 @@ plugins:
         util: 'TES4Edit v4.0.4b'
 
 ###### AudioVisual - Sounds & Music ######
+  - name: '(AgarMoreVariedSpellEffects|AgarVariedSpellSounds_SupremeMagicka)\.esp'
```

The regex can be simplified a bit:

```suggestion
  - name: 'Agar(More)?VariedSpell(Effects|Sounds_SupremeMagicka)\.esp'
```